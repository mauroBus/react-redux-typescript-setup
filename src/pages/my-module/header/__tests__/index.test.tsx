import { shallow } from 'enzyme'
import React from 'react'
import { Header } from '../'

describe('MyModule::Header', () => {
  it('renders as a "header" element', () => {
    const tree = shallow(<Header />)
    expect(tree.type()).toBe('header')
  })

  it('renders text message', () => {
    const tree = shallow(<Header />)
    expect(tree.text()).toBe('This is a header.')
  })

  it('renders', () => {
    const tree = shallow(<Header />)
    expect(tree).toMatchSnapshot()
  })
})
