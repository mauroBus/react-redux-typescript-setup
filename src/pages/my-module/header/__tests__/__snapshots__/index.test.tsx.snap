// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyModule::Header renders 1`] = `
<header
  className="header"
>
  This is a header.
</header>
`;
